js/ui.js

```diff
@@ -25,6 +26,30 @@ function display(data) {
 	setErrorVisible(false)
 }
 
+function displayCommunesFetchResults(info, values) {
+	// Inform the user
+	var label = document.querySelector('label[for="depcom_entreprise"]')
+	label.innerHTML = info || ''
```

Comment:
> Use `textContent` rather than `innerHTML` if you don't need to actually HTML parser to avoid incurring the [performance cost](https://jsperf.com/read-innerhtml-vs-innertext-vs-nodevalue-vs-textcontent/5).